moviebot/nlu/neural_nlu.py

```diff
@@ -173,3 +189,4 @@ class DS:
         )
 
         print([str(da) for da in da])
+    """
```

Comment:
This entire block (`if __name__ == "__main__"`) should be removed. Use unit testing instead.